cardano-git-rev/src/Cardano/Config/Git/RevFromGit.hs

```diff
@@ -0,0 +1,32 @@
+module Cardano.Config.Git.RevFromGit (
```

Comment:
Maybe this module name should change?

Reply:
I agree.  The module name should change.

Reply:
Changed the module name.